packages/email/emails/survey/embed-survey-preview-email.tsx

```
@@ -0,0 +1,29 @@
+  html,
+  environmentId,
+}: EmbedSurveyPreviewEmailProps): React.JSX.Element {
+  return (
+    <EmailTemplate>
+      <Container>
+        <Heading>Preview Email Embed</Heading>
+        <Text>This is how the code snippet looks embedded into an email:</Text>
+        <Text className="text-sm">
+          <b>Didn&apos;t request this?</b> Help us fight spam and forward this mail to hola@formbricks.com
+        </Text>
+        <div dangerouslySetInnerHTML={{ __html: html }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)